public/less/_repository.less

```diff
@@ -1326,3 +1326,36 @@
 		width: 100%!important;
 	}
 }
+
+.tab-size-1 {
```

Comment:
Something like this? https://github.com/andreynering/gogs/blob/460907f09fc88cf9e67b095e298e2f4037ddd19b/public/less/_base.less#L369